stock_operating_unit/data/stock_data.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016-2017 Eficent Business and IT Consulting Services S.L.
+     Copyright 2016-2017 Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo noupdate="1">
+
+    <record id="stock.warehouse0" model="stock.warehouse">
+        <field name="operating_unit_id" ref="operating_unit.main_operating_unit"/>
```

Comment:
Here is assigning "Your Company" warehouse to the Main OU. The other warehouses remain without any OU.

Reply:
Yes. The warehouse (not the store ...) that is assigned to the Main Organizational Unit is the only one that is not shown. But I do not see any error in code or difference in the database, as I said before.

That's why I think some permission from another module may be overwriting me that permission rule.

To clarify. I do not have to select the Warehouse even in the user permissions. I do not think the problem is in these rules that we are commenting, since they read permissions from that configuration. And there, the Warehosue with Main OU does not appear to select.

I attach screenshots:
**Operating Units:** https://snag.gy/QJtxbs.jpg
**Warehouses:** https://snag.gy/MvXVEh.jpg
**Main OU:** https://snag.gy/9Wzplq.jpg
**Assign OUs to user screen:** https://snag.gy/ohNA5a.jpg